### PR TITLE
fix types

### PR DESCRIPTION
**Pull request by one-more**

### CODE REVIEW

File: src/components/x-clock/x-clock-react.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import * as React from "react";
 import {Component} from "@micro-frontends/core";
 import {renderReact} from "@micro-frontends/framework";
 
-export class XClockReact extends Component {
+export class XClockReact extends Component<undefined, undefined> {
```
Comment (nickovchinnikov) [inline src/components/x-clock/x-clock-react.tsx]: Component<undefined, undefined> would it be default values?

**Review by nickovchinnikov — APPROVED** [review]

File: src/components/x-clock/x-clock-react.tsx (hunk 1 above)
Comment (one-more) [reply src/components/x-clock/x-clock-react.tsx]: fixed to default values

File: src/store.ts [hunk 2]
```diff
@@ -1,25 +1,8 @@
...
+import {createStore, combineReducers} from "redux";
 import {xChatReducers} from "~/modules/x-chat";
 
 export const store = createStore(
     combineReducers({
         ...xChatReducers,
     })
-);
-
-export function subscribeWithSelector(selector: Function, cb: Function): Unsubscribe {
```
Comment (nickovchinnikov) [inline src/store.ts:10]: is it ok?

**Review by nickovchinnikov — APPROVED** [review]

File: src/store.ts (hunk 2 above)
Comment (one-more) [reply src/store.ts:10]: it's not needed since it's in @micro-frontends/core package

**Pull request merged**